CHANGES.rst

```diff
@@ -21,7 +21,6 @@ resample
 
 pipeline
 --------
-
```

Comment:
This is causing the CI failure. There's no need to change this file anymore so the CI checks that no commits touch it.

Reply:
Thanks, Brett!